packages/react/src/themes/base/components/Checkbox/checkboxStyles.ts

```diff
@@ -30,9 +30,9 @@ const checkboxStyles: ComponentSlotStylesInput<CheckboxProps & CheckboxState, Ch
     padding: v.checkboxPadding,
 
     ...(p.checked && {
-      background: v.checkedCheckboxBackground,
-      borderColor: v.checkedCheckboxBorderColor,
-      color: v.checkedCheckboxColor,
+      background: v.checkboxCheckedBackground,
```

Comment:
Not sure why the name of these were changed. More over, not sure why we have the word `checkbox` in any variable of the `Checkbox` component. We may simplify this to `checkedBackground`